tests/test_extremes.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import pandas as pd
+import pooch
+
+from xhydro.extreme_value_analysis.parameterestimation import (
+    gevfit,
+    gevfitbayes,
+    gevfitpwm,
+    gpfit,
+    gpfitbayes,
+    gpfitpwm,
+    gumbelfit,
+    gumbelfitbayes,
+    gumbelfitpwm,
+)
+
+# Dataset taken from tests in Extremes.jl
+GITHUB_URL = "https://github.com/hydrologie/xhydro-testdata"
+BRANCH_OR_COMMIT_HASH = "main"
+
+genextreme_data = pooch.retrieve(
+    url=f"{GITHUB_URL}/raw/{BRANCH_OR_COMMIT_HASH}/data/extreme_value_analysis/genextreme.zip",
+    known_hash="md5:cc2ff7c93949673a6acf00c7c2fac20b",
+    processor=pooch.Unzip(),
+)
+
+genpareto_data = pooch.retrieve(
+    url=f"{GITHUB_URL}/raw/{BRANCH_OR_COMMIT_HASH}/data/extreme_value_analysis/genpareto.zip",
+    known_hash="md5:ecb74164db4bbfeabfc5e340b11e7ae8",
+    processor=pooch.Unzip(),
+)
+
+GEV_NONSTATIONARY = pd.read_csv(genextreme_data[0])
+GEV_STATIONARY = pd.read_csv(genextreme_data[1])
+GP_NONSTATIONARY = pd.read_csv(genpareto_data[0])
+GP_STATIONARY = pd.read_csv(genpareto_data[1])
```

Comment:
This is good. I might just move it into `conftest.py` and convert it into a set of pytest fixtures. Otherwise, every process that examines this file will be fetching/reading data. Would that work for you?

Reply:
All right, will look into it once I fix the tests that block the pipeline

Reply:
@Zeitsperre I changed the test data. I hardcode the data used in Extremes.jl into the test file to avoid converging problems in some tests. should I move this data into conftest.py?